.github/workflows/pkgci_shark_ai.yml

```
@@ -25,17 +25,25 @@ jobs:
     strategy:
       matrix:
         version: [3.11]
+        test_device: ["cpu", "gfx90a", "gfx942"] # temporarily not running on "gfx942" due to contention with other ci tasks
       fail-fast: false
-    runs-on: azure-cpubuilder-linux-scale
-    # runs-on: ubuntu-latest # everything else works but this throws an "out of resources" during model loading
-    # TODO: make a copy of this that runs on standard runners with tiny llama instead of a 8b model
+    runs-on: >-
+      ${{
+        matrix.test_device == 'cpu' && 'azure-cpubuilder-linux-scale' ||
+        matrix.test_device == 'gfx942' && 'linux-mi300-gpu-1' ||
+        matrix.test_device == 'gfx90a' && 'nodai-amdgpu-mi250-x86-64'
+      }}
     defaults:
       run:
         shell: bash
     env:
       PACKAGE_DOWNLOAD_DIR: ${{ github.workspace }}/.packages
       VENV_DIR: ${{ github.workspace }}/.venv
     steps:
+      - name: Run rocminfo
+        continue-on-error: true
+        run: |
+          rocminfo || echo "rocminfo not available"
```

Only run on GPU runners instead of using `continue-on-error`?

```suggestion
      - name: Run rocminfo
        if: contains(matrix.test_device, 'gfx')
        run: rocminfo
```